app/dashboard/views/support.py

```diff
@@ -0,0 +1,98 @@
+import json
+import urllib.parse
+from typing import Union
+
+import requests
+from flask import render_template, request, flash, url_for, redirect
+from flask_login import login_required, current_user
+from werkzeug.datastructures import FileStorage
+
+from app.dashboard.base import dashboard_bp
+from app.log import LOG
+from app.models import Mailbox
+from app.config import ZENDESK_HOST
+
+VALID_MIME_TYPES = ['text/plain', 'message/rfc822']
+
+
+@dashboard_bp.route("/support", methods=["GET"])
+@login_required
+def show_support_dialog():
+    if not ZENDESK_HOST:
+        return render_template("dashboard/support_disabled.html")
+    return render_template("dashboard/support.html", ticket_email=current_user.email)
+
+
+def check_zendesk_response_status(response_code: int) -> bool:
+    if response_code != 201:
+        if response_code in (401 or 422):
+            LOG.debug('Could not authenticate')
+        else:
+            LOG.debug('Problem with the request. Status {}'.format(response_code))
```

Comment:
⬆️

Reply:
If it's the same. `debug` is more explicit and easier to read.

Reply:
`LOG.d` is the widely used shortcut, no other place in the codebase uses `LOG.debug`, `LOG.e` is actually short for `exception` instead of `error`.